addons/account_check_printing/data/check_printing.xml

```diff
@@ -15,8 +15,8 @@
             <field name="name">Print Checks</field>
             <field name="model_id" ref="account.model_account_payment"/>
             <field name="code">
-                if context.get('active_model') == 'account.payment' and context.get('active_ids'):
-                    action = self.pool['account.payment'].print_checks(cr, uid, context['active_ids'], context=context)
+if context.get('active_model') == 'account.payment' and context.get('active_ids'):
+    action = self.pool['account.payment'].print_checks(cr, uid, context['active_ids'], context=context)
```

Comment:
Properly indenting code is perfectly valid as far as we're concerned, there's no reason to change that. Dedent snippets before parsing them instead.